### PR TITLE
UI for the Organisation Tab

### PR DESCRIPTION
**Pull request by DevanshCodes**
Have completed the UI for the Organisation Tab according to the Figma Design. Used Material UI for the implementation, also not hard coded the org-data instead used states and props for the same for easier back-end implementation.

Fixes issue #386 

Organisation Page

### CODE REVIEW

**Comment by DevanshCodes** [comment]
> @DevanshCodes Just add org logo above the user edit button and also improve the alignment of user edit button.
> Everything else looks good!
> Nice work!
> 

Okay cool, I'll do that.

**Comment by DevanshCodes** [comment]
Added the OrgImage and fixed the button!

**Comment by tanujvyas10** [comment]
@DevanshCodes Please update your Gif since u added the logo too

**Comment by DevanshCodes** [comment]
> @DevanshCodes Please update your Gif since u added the logo too
> 

Hey, you can check the latest commit by simply visiting the netlify build https://deploy-preview-390--donut-frontend-r.netlify.com/organization. Link is also given above in Netlify comment.

File: src/user/organization/org-contact/org-contact.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                    .avatar {
+                        width: 25px;
+                        height: 25px;
+                        margin-right: 5px;
+                    }
+                }
+            }
+        }
+    }
+}
```
Comment (vaibhavdaren) [inline src/user/organization/org-contact/org-contact.scss:39]: nice work

**Pull request closed without merging**